dhall/src/Dhall/DirectoryTree.hs

```diff
@@ -54,8 +55,12 @@ import qualified Prettyprinter               as Pretty
 import qualified Prettyprinter.Render.String as Pretty
 import qualified System.Directory            as Directory
 import qualified System.FilePath             as FilePath
+#ifdef mingw32_HOST_OS
```

Comment:
This is exactly the condition used by `unix-compat` prior to 0.7: https://github.com/haskell-pkg-janitors/unix-compat/commit/87ffe0554e85875780cedca610a9accc9c23cd4c#diff-93ed634b109420481984ea0fecc52a958c1d98c2728f8ec0b4c6ed741a8d4a24L36